includes/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper.php

```
@@ -16,26 +16,441 @@
+				)
+			);
+		}
+
+		$this->map_address(
+			$order,
+			$shipping_address,
+			$session->get_shipping_name(),
+			$session->get_shipping_phone(),
+			'shipping'
+		);
+	}
```

PHPMD (2.15.0)

[error] The method map_addresses is not named in camelCase. (undefined)

(CamelCaseMethodName)